docs/sources/panels/configure-data-links/index.md

```diff
@@ -66,9 +66,23 @@ Value-specific variables are available under `__value` namespace:
 
 When linking to another dashboard that uses template variables, select variable values for whoever clicks the link.
 
-`${myvar:queryparam}` - where `myvar` is a name of the template variable that matches one in the current dashboard that you want to use.
+- `${myvar:queryparam}` - where `myvar` is the name of the template variable that matches a variable in the dashboard you want to use.
 
-If you want to add all of the current dashboard's variables to the URL, then use `__all_variables`.
+| State        | Result                                                                        |
+| ------------ | ----------------------------------------------------------------------------- |
+| one selected | `value`                                                                       |
+| multiple     | escaped html `{value1,valuen}`                                                |
+| All          | the allValue with no surrounding symbols such as `.+` if that is the allValue |
```

Comment:
```suggestion
| `All`          | The `allValue` with no surrounding symbols, such as `.+` if `.+` is the `allValue` |
```
* The other states are lowercase, and this one is uppercase. Is this intentional?
* I tried to clarify this definition, whose definition is unclear. What does `All` mean? What does it do?